src/main/java/org/amahi/anywhere/adapter/RecentFilesAdapter.java

```diff
@@ -129,7 +142,8 @@ public void tearDownCallbacks() {
 
     @Override
     public int getItemCount() {
-        return recentFiles.size();
+        int SHIMMER_ITEM_NUMBER = 20;
+        return showShimmer ? SHIMMER_ITEM_NUMBER : recentFiles.size();
```

Comment:
Personal opinion. I hate tertiary operators. Completely destroys readability.

Reply:
Also, you can move `SHIMMER_ITEM_NUMBER` to `Constants.java` so that all constant variables are present at one place.

Reply:
Done